### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

Main goal: fix broken pipeline as jackson 2.9.10 is marked as vulnerable by owasp.

Latest Spring Boot (2.2.0) has already migrated to jackson 2.10.0

Updating other dependencies along way :) - sample app

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java [hunk 1]
```diff
@@ -160,7 +160,7 @@ void should_return_form_not_found_exception_when_form_type_case_does_not_match()
         MvcResult mvcResult = mockMvc
             .perform(
                 post("/forms/Personal/validate-ocr") //only PERSONAL is valid form type
-                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                    .contentType(APPLICATION_JSON_VALUE)
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java]: are changes is this file related to dependencies update?

Comment (doncem) [reply src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java]: Yes. Usage of `APPLICATION_JSON_UTF8` is deprecated in Spring Boot 2.2. Only `APPLICATION_JSON_VALUE` is relevant to this class so moving to static import instead
